tests/components/nina/test_binary_sensor.py

```diff
@@ -155,7 +155,7 @@ async def test_sensors_without_corona_filter(hass: HomeAssistant) -> None:
             == "Corona-Verordnung des Landes: Warnstufe durch Landesgesundheitsamt ausgerufen"
         )
         assert state_w1.attributes.get(ATTR_ID) == "mow.DE-BW-S-SE018-20211102-18-001"
-        assert state_w1.attributes.get(ATTR_SENT) == "2021-11-02T20:07:16+01:00"
+        assert state_w1.attributes.get(ATTR_SENT) == "2021-11-02T19:07:16+00:00"
         assert state_w1.attributes.get(ATTR_START) == ""
```

Comment:
We don't set empty strings in state attributes. Either remove the item completely if the value is unknown, which is our current standard, or set a None value, which we'll probably change our standard to.